src/Queue/Queue.php

```diff
@@ -4,58 +4,202 @@
 
 namespace Baka\Queue;
 
+use function Baka\envValue;
 use Phalcon\Di;
+use Phalcon\Security\Random;
 use PhpAmqpLib\Message\AMQPMessage;
+use PhpAmqpLib\Wire\AMQPTable;
 
 class Queue
 {
     /**
      * default canvas queues system name.
      */
-    const EVENTS = 'events';
-    const NOTIFICATIONS = 'notifications';
-    const JOBS = 'jobs';
+    public const EVENTS = 'events';
+    public const NOTIFICATIONS = 'notifications';
+    public const JOBS = 'jobs';
+    public const EXCHANGE_PREFIX = 'baka_exchange';
+    public const EXCHANGE_TYPE = 'direct';
 
-    public static bool $passive = false;
-    public static bool $durable = true;
-    public static bool $exclusive = false;
-    public static bool $auto_delete = false;
+    private static bool $passive = false;
+    private static bool $durable = true;
+    private static bool $exclusive = false;
+    private static bool $auto_delete = false;
+    private static bool $noWait = false;
 
     /**
-     * Send a msg to Queue.
+     * Dechare an exchange on the channel.
      *
      * @param string $name
-     * @param array|object|mixed $msg
+     * @param string $type
+     * @param bool $force
      *
-     * @return bool
+     * @return void
      */
-    public static function send(string $name, $msg) : bool
+    public static function declareExchange(string $name, bool $force = false) : void
```

Comment:
@PaulMejiaFeliz add the description or logic to the pull or code for future read https://engineering.nanit.com/rabbitmq-retries-the-full-story-ca4cc6c5b493

Reply:
I just added it to the doc of the `createFlowWithDelay` method